ITISFoundation/osparc-simcore
# 🎨 [Frontend] Request Account form for oSPARC 🚨

## What do these changes do?

If product vendor's information says that ``invitation_form`` and the login configuration says ``invitation_required``, a Request Account form will be displayed (no more ``is_product_billable`` checks for this).

![RequestAccount](https://github.com/user-attachments/assets/f926bdd3-0535-4fa2-884b-ff32edee3f8f)


## Related issue/s

closes https://github.com/ITISFoundation/osparc-issues/issues/1584


## How to test

- set  `"invitation_form": true` in database (see below)
- open product's url
- click invitation
- you should get the form
- repeat with `"invitation_form": false` to obtain either a link (if `invitation_url` is provided) or an email (if not)

## Dev-ops checklist 🚨

- 🗃️  Add `invitation_form: true` in `products.vendor` column of the database. e.g. 
```json
{
    "url": "https://acme.com",
    "license_url": "https://acme.com/license",
    "invitation_form": true,
    "name": "ACME",
    "copyright": "© ACME correcaminos",
    "has_landing_page": false
}
```
